packages/contracts-bedrock/test/L1/OPStackManager.t.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+// Testing utilities
+import { Test } from "forge-std/Test.sol";
+
+// Target contract
+import { OPStackManager } from "src/L1/OPStackManager.sol";
+
+// Exposes internal functions for testing.
+contract OPStackManager_Harness is OPStackManager {
+    function chainIdToBatchInboxAddress_exposed(uint256 l2ChainId) public pure returns (address) {
+        return super.chainIdToBatchInboxAddress(l2ChainId);
+    }
+}
+
+// Unlike other test suites, we intentionally do not inherit from CommonTest or Setup. This is
+// because OPStackManager acts as a deploy script, so we start from a clean slate here and
+// work OPStackManager's deployment into the existing test setup, instead of using the existing
+// test setup to deploy OPStackManager.
+contract OPStackManager_Test is Test {
+    OPStackManager opsm;
+
+    // Default dummy parameters for the deploy function.
+    OPStackManager.Roles roles;
+    uint256 l2ChainId = 1234;
+    uint32 basefeeScalar = 1;
+    uint32 blobBasefeeScalar = 1;
+
+    function setUp() public {
+        opsm = new OPStackManager();
+    }
+}
+
+contract OPStackManager_Deploy_Test is OPStackManager_Test {
+    function test_RevertsIf_L2ChainIdEqualsZero() public {
```

Comment:
The test function names are close but don't quite follow the convention defined in the Style Guide. 
They are also checked with `pnpm lint:forge-tests:check` but that evidently does not run in CI and many of our tests fail it.

Reply:
Ah, thank you for flagging! I haven't done tons of smart contract dev in this repo so I will review the style guide.

> They are also checked with `pnpm lint:forge-tests:check` but that evidently does not run in CI and many of our tests fail it.

Oh wow, we do have a lot of failures there. So maybe I won't add that to our CI and fix existing failures yet... 😅